src/modules/app/createReduxStore.js

```diff
@@ -0,0 +1,52 @@
+import { applyMiddleware, combineReducers, compose, createStore } from 'redux'
```

Comment:
⚠️ Probably will conflict with @klinzo s PR. Because he depends on Store.js

Reply:
I've already merged that in. I could refactor Steffs dependency to use mockStore

Reply:
@klinzo s task does not work with a mockStore

Reply:
@klinzo k if it works ok, but wasn't this useful for the test with the async stuff and real mock?

Reply:
it also works with a mock store, and we have a separate task for this (WEBAPP-167), but we should talk about this today on ts